### PR TITLE
Modified docs/Documentation/03a-stoplight-flavored-markdown.md

### PR DESCRIPTION
**Pull request by mnaumanali94**

### CODE REVIEW

File: docs/Documentation/03a-stoplight-flavored-markdown.md [hunk 1]
```diff
@@ -1,128 +1,136 @@
-# Stoplight Flavored Markdown
+# Stoplight Flavored Markdown (SMD)
 
-Stoplight Flavored Markdown (or SMD, for short) with two guiding principles:
+## Overview
 
--   It must be **human readable**, where the raw Markdown source can be viewed and edited using standard, readily available text editors.
--   It must **degrade gracefully**, where rendering Stoplight Flavored Markdown _outside_ of Stoplight should still be usable, clean, and, most importantly, readable.
+Stoplight flavored markdown can be used to enhance existing markdown documentation with necessary documentation components like code blocks, images, diagrams e.t.c.
```
Comment (paulatulis) [inline docs/Documentation/03a-stoplight-flavored-markdown.md:5]:
```suggestion
Stoplight Flavored Markdown can be used to enhance existing Markdown documentation with necessary documentation components like code blocks, images, diagrams etc.
```

File: docs/Documentation/03a-stoplight-flavored-markdown.md [hunk 2]
```diff
@@ -1,128 +1,136 @@
...
 
-## Block Types
+1.  SMD is human readable. A human with a simple text editor can easily read and write SMD.
+2.  SMD degrades gracefully. SMD documents rendered on `github.com` should be clean and readable.
 
-### Callouts
+### The Approach
 
-A callout is a Markdown block quote with an optional annotation that indicates the theme of the callout and intention of the message. The block annotation for a callout can include:
+1.  Stoplight flavored markdown extends GitHub flavor markdown with inline comment annotations.
```
Comment (paulatulis) [inline docs/Documentation/03a-stoplight-flavored-markdown.md:27]:
```suggestion
1.  Stoplight Flavored Markdown extends GitHub Flavored Markdown with inline comment annotations.
```

File: docs/Documentation/03a-stoplight-flavored-markdown.md [hunk 3]
```diff
@@ -1,128 +1,136 @@
...
+2.  The value inside of the annotations is a YAML object, and the annotation affects the markdown block that directly follows it in the document.
```
Comment (paulatulis) [inline docs/Documentation/03a-stoplight-flavored-markdown.md:28]:
```suggestion
2.  The value inside of the annotations is a YAML object, and the annotation affects the Markdown block that directly follows it in the document.
```

File: docs/Documentation/03a-stoplight-flavored-markdown.md [hunk 4]
```diff
@@ -1,128 +1,136 @@
...
 
--   `theme` (`string`), which must be one of the following values:
-    -   `info`
-    -   `success`
-    -   `warning`
-    -   `danger`
+By leveraging comments to store annotations, Stoplight flavored markdown degrades gracefully to any other markdown
```
Comment (paulatulis) [inline docs/Documentation/03a-stoplight-flavored-markdown.md:30]:
```suggestion
By leveraging comments to store annotations, Stoplight Flavored Markdown degrades gracefully to any other Markdown
```

File: docs/Documentation/03a-stoplight-flavored-markdown.md [hunk 5]
```diff
@@ -1,128 +1,136 @@
...
+renderer (Github, for example).
```
Comment (paulatulis) [inline docs/Documentation/03a-stoplight-flavored-markdown.md:31]:
```suggestion
renderer (GitHub, for example).
```

File: docs/Documentation/03a-stoplight-flavored-markdown.md [hunk 6]
```diff
@@ -1,128 +1,136 @@
...
+- [x] two
+- [ ] three
+```
 
-A SMD code block is a Markdown code fence with an optional YAML annotation used to tweak the presentation of the code block. The annotation can include the following attributes:
+## Code Blocks
 
--   `title` (`string`), which is a string title for the resulting block
--   `lineNumbers` (`true`\|`false`), which denotes whether line numbers should be included (defaults to `true`)
+A SMD code block is an MD code fence with an optional annotation to tweak the presentation of the code block.
```
Comment (paulatulis) [inline docs/Documentation/03a-stoplight-flavored-markdown.md:115]:
```suggestion
An SMD code block is an MD code fence with an optional annotation to tweak the presentation of the code block.
```

File: docs/Documentation/03a-stoplight-flavored-markdown.md [hunk 7]
```diff
@@ -131,104 +139,220 @@ function fibonacci(num){
...
-| col 2 is      |   centered    |   \$12 |
-| zebra stripes |   are neat    |    \$1 |
+![Studio project share](https://i.imgur.com/ueOOL8X.png)
 ```
 
-Results in:
+## Graphs and Diagrams
 
-<!-- title: My Table Title -->
+You can in graphs and diagrams in your markdown via the `mermaid` code block language tag. Here are some examples:
```
Comment (paulatulis) [inline docs/Documentation/03a-stoplight-flavored-markdown.md:272]: You can what?

Comment (paulatulis) [reply docs/Documentation/03a-stoplight-flavored-markdown.md:272]:
```suggestion
You can in graphs and diagrams in your Markdown via the `mermaid` code block language tag. Here are some examples:
```

File: docs/Documentation/03a-stoplight-flavored-markdown.md [hunk 8]
```diff
@@ -131,104 +139,220 @@ function fibonacci(num){
...
+      Go downstairs: 5: Me
+      Sit down: 5: Me
+```
+
+
+## JSON Schema
 
-The example above results in:
+The SMD JSON schema block is an MD code block with an additional `json_schema` language tag. The contents of the code
+fence should be the JSON schema object to be rendered. The primary language tag can be `YAML, `YML`, or `JSON.
```
Comment (paulatulis) [inline docs/Documentation/03a-stoplight-flavored-markdown.md:355]:
```suggestion
fence should be the JSON schema object to be rendered. The primary language tag can be `YAML, `YML`, or `JSON`.
```

File: docs/Documentation/03a-stoplight-flavored-markdown.md [hunk 9]
```diff
@@ -251,132 +375,109 @@ The example above results in:
...
 <!-- type: tab-end -->
+```
 
-<!-- theme: warning -->
-
-> Tab container start and end annotations are **required**, and cannot currently be nested.
+## HTML
 
-### HTTP Request Maker
+Most basic HTML is supported. **However, we highly recommend using the markdown equivalent whenever possible.
```
Comment (paulatulis) [inline docs/Documentation/03a-stoplight-flavored-markdown.md:447]:
```suggestion
Most basic HTML is supported. **However, we highly recommend using the Markdown equivalent whenever possible.
```

File: docs/Documentation/03a-stoplight-flavored-markdown.md [hunk 10]
```diff
@@ -251,132 +375,109 @@ The example above results in:
...
-```
+## Embeds
 
 <!-- theme: warning -->
 
-> In the example above, be sure to remove the `\` that precedes the last three backticks at the end of the `yaml` code fence before using.
+> This hasn't been implemented yet. **[Vote for this feature](https://roadmap.stoplight.io/c/28-rich-embeds-such-as-code-samples-videos-etc)!**
 
-The Markdown above results in:
+This is a youtube video:
```
Comment (paulatulis) [inline docs/Documentation/03a-stoplight-flavored-markdown.md:473]:
```suggestion
This is a YouTube video:
```

**Review by paulatulis — CHANGES_REQUESTED** [review]

**Comment by paulatulis** [comment]
@mnaumanali94 Just curious why sometimes we treat 'Stoplight Flavored Markdown' as a proper noun and sometimes not. I was under the impression it should be similar to how GitHub refers to GitHub Flavored Markdown. Also, I believe Markdown is a proper noun in general?

**Comment by mnaumanali94** [comment]
Great catches @paulatulis. Fixed them!

File: docs/Documentation/03a-stoplight-flavored-markdown.md (hunk 6 above)
Comment (paulatulis) [reply docs/Documentation/03a-stoplight-flavored-markdown.md]: @mnaumanali94 if you're going to say "an MD", it should also be "An SMD".

File: docs/Documentation/03a-stoplight-flavored-markdown.md (hunk 8 above)
Comment (paulatulis) [reply docs/Documentation/03a-stoplight-flavored-markdown.md]: Did you mean to resolve this? It looks strange if you don't add the extra ` after JSON

File: docs/Documentation/03a-stoplight-flavored-markdown.md (hunk 10 above)
Comment (paulatulis) [reply docs/Documentation/03a-stoplight-flavored-markdown.md]: I am not sure if there's a reason it's resolved. Maybe by mistake!

File: docs/Documentation/03a-stoplight-flavored-markdown.md (hunk 5 above)
Comment (paulatulis) [reply docs/Documentation/03a-stoplight-flavored-markdown.md:31]: GitHub

**Review by paulatulis — CHANGES_REQUESTED** [review]
I think that the conversations were resolved, but they didn't change, so I unresolved them. Let me know if there's anything you disagree with!

**Comment by mnaumanali94** [comment]
> I think that the conversations were resolved, but they didn't change, so I unresolved them. Let me know if there's anything you disagree with!

Ugh no. Wanted to commit them all. I think just clicking commit multiple times didn't work.

**Review by paulatulis — COMMENTED** [review]
@mnaumanali94 so sorry a couple of things slipped by when I looked before. Can you just commit/double check those and then I think it's g2g!

**Review by paulatulis — APPROVED** [review]
Approving! Note to selves: add a little padding on left-hand side :)

**Pull request merged**